bindings/python/examples/how_tos/accounts_and_addresses/create_account.py

```diff
@@ -28,8 +28,7 @@
     raise Exception(".env MNEMONIC is undefined, see .env.example")
 
 # Store the mnemonic in the Stronghold snapshot, this only needs to be done once
-wallet.store_mnemonic(
-    os.environ['MNEMONIC'])
+# wallet.store_mnemonic(os.environ['MNEMONIC'])
```

Comment:
🤔

Reply:
Uncommented again
Disabled it during testing so I don't have to clean the db or create another example, but maybe we should split wallet and account creation